core/common/src/main/java/alluxio/underfs/UnderFileSystem.java

```diff
@@ -511,6 +511,15 @@ public int getValue() {
    */
   AlluxioURI resolveUri(AlluxioURI ufsBaseUri, String alluxioPath);
 
+  /**
+   * Sets the access control list of a file or directory in under file system.
```

Comment:
If the UFS does not support acls, what is supposed to happen? Exception, or noop?

Reply:
edited comments to clarify